exp/api/v1alpha4/azuremanagedmachinepool_types.go

```diff
@@ -42,6 +42,9 @@ type AzureManagedMachinePoolSpec struct {
 	// +kubebuilder:validation:Enum=System;User
 	Mode string `json:"mode"`
 
+	// AvailabilityZones - Availability zones for nodes. Must use VirtualMachineScaleSets AgentPoolType.
+	AvailabilityZones *[]string `json:"availabilityZones,omitempty"`
```

Comment:
To copy-paste @alexeldeib's nit from AAD PR, maybe just a slice instead of pointer to a slice could do the job here.

Reply:
ack